src/main/res/values/arrays.xml

```diff
@@ -574,4 +574,16 @@
 		<item>url</item>
 	</string-array>
 
+	<string-array name="pref_behaviour_nsfw">
+		<item>No NSFW Content</item>
```

Comment:
These should be defined at the end of strings.xml to allow for translation :)

Also I'd change the option text to "Block", "Allow", and "Only show NSFW".